### PR TITLE
Update for Feature 1

### PR DESCRIPTION
**Pull request by liannivin**

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -2,6 +2,9 @@ body {
   /* background-image: url("/img/bg.jpg"); */
   background-color: #eaeaea;
 }
+textarea {
+  resize: none;
```
Comment (deyviconde) [inline public/css/style.css]: ✌ great !!!

File: views/layout.erb [hunk 2]
```diff
@@ -33,7 +33,8 @@
             </div>
             <div class="col-sm-8">
             <!-- Daily -->
-              <%= yield_content :daily %>
+              <%= yield_content :daily if content_for?(:daily) %>
```
Comment (deyviconde) [inline views/layout.erb]: Great, this help us to show a customized view

File: route.rb [hunk 3]
```diff
@@ -13,6 +13,21 @@
...
+post '/' do 
+  #save_workshop(params["title"], params["content"])
+  inputs = Hash.new 
+  inputs["title"]= params["title"]
+  inputs["datetime"]= params["date"]+params["time"]
+  inputs["content"]= params["content"]
+  inputs["content_before"]= [] 
+  inputs["highlight"]= 0
+  inputs["is_deleted"]= 0
+  inputs["deleted_datetime"]= ""
```
Comment (Linzeur) [inline route.rb]: It is not mandatory but for presentation it's better that before and after the equal sign there is a space. It is also for plus sign in the line 20.

**Comment by deyviconde** [comment]
Feature 1 progress ! ✌

**Pull request merged**